### PR TITLE
Fix - Results page in web application showing 0 for all scores on event results admin. 

### PR DESCRIPTION
**Pull request by davidsmonkeys**
## Description

In the process of upgrading the API to .NET 8 we also updated the version of EF Core. In doing that we came accross an error called fixup. In addressing this problem we dropped the end of the query for getting the results which was the part that had the score in it. 

Reverting this change with a the exception of not reattaching the second include the the venue solves the problem. 

We've also corrected a spelling error. 

Fixes #25 (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

We have tested this locally and see the results return to the results page

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules 
### :bulb: PR Summary generated by FirstMate

**Overview**: Fixed the results page displaying zero scores for events in the admin panel.

**Changes**:
*Configuration Updates:*
- Corrected spelling of "Environment" in `appsettings.json` for both WebClient and PwaClient.

*Data Retrieval Enhancements:*
- Added inclusion of `DanceScores` and its `ScoreCategory` in `DanceRepository.cs` to ensure scores are fetched correctly.
- Adjusted comments to clarify changes related to EF Core 8 upgrade and its impact on data retrieval.

**TLDR**: The PR fixes the results page by ensuring that dance scores are included in the data retrieval process, addressing the issue of displaying zero scores. Review the changes in `DanceRepository.cs` for the main functionality update.

<sub><sup>Generated by FirstMate and automatically updated on every commit.</sup></sub>

### CODE REVIEW

**Review by dertinfo — APPROVED** [review]

**Comment by davidsmonkeys** [comment]
After the merge of this change into the main branch and the pipelines run to deploy it to the staging environment I had a issue 500.13 with the staging environment (that was functioning yesterday) due to the change. After attempting to restart the service and then redeploy the previou code the error persisted. 

I managed to resolve the issue by proceeding to the app service kudu and upgrading and then reinstalling the  ASP.NET Core 8.0 (x86) Runtime. It's unclear why this would have been a problem but has resolved the issue.

**Comment by davidsmonkeys** [comment]
On promoting this change to live we had exactly the same issue and had to apply the kudu upgrade to the .NET runtime from version 8.0.8 to 8.0.10. for package ASP.NET Core 8.0 (x86) Runtime and then 9.0.0-rc-2-24474-3 for  logging. 

However this doesn't appear to have succeeded in the first  instance. We not have a production incident. 

We enable the logs and try to find the problem and after a short amount of time the site comes back up. 

I suspect that when using kudu to update the .NET minor version then it takes longer then the feedback suggests in the kudu interface and is why the application remains down. 

In hindsight I should probably updated the app service first with the previous version of the code then authorised the deployment.
